### DISCUSSION TITLE
Automatic benchmarking

### DISCUSSION THREAD

**Discussion by arthuro555** [4ian/GDevelop#3361] [Enhancement ideas (and their technical discussions)]
I think it would be great to have some automatic benchmarks run via the CI on every PR and master commit, with comparisons to the latest commit on master that isn't the one being benchmarked. 
This would allow to spot performance regressions early, and to help evaluate the impact of some optimizations.

**Comment by 4ian** [comment]
Totally, I would be very interested to have this. Not sure what are the best practices, I'v seen things like a table like this: https://fast.vlang.io/

Would be great to surface somewhere the existing benchmarks :)
